circom-prover/Cargo.toml

```diff
@@ -30,9 +26,18 @@ arkworks = [
     "ark-bls12-381",
     "ark-poly",
 ]
-rapidsnark = []
+rapidsnark = [
+    "rust-rapidsnark",
+    "serde_json",
+    "ark-bn254",
+    "ark-ec",
+    "ark-groth16",
+    "ark-bls12-381",   # TODO: remove this
```

Comment:
Is it a leftover or will fix it in a later PR? If we plan to fix it in a later PR, we can put issue number here.

Reply:
I am still thinking about if this conversion is necessary.
but we can create an issue to discuss this.
https://github.com/zkmopro/mopro/issues/329

Reply:
[d51841a](https://github.com/zkmopro/mopro/pull/318/commits/d51841a5d61fefe0973e81c3195398664d61ca56)